crates/hc_demo_cli/src/demo.rs

```diff
@@ -40,7 +40,7 @@ impl RunOpts {
 }
 
 /// The default configured signal server url.
-pub const DEF_SIGNAL_URL: &str = "wss://signal.holo.host";
+pub const DEF_SIGNAL_URL: &str = "wss://sbd.holo.host";
```

Comment:
woops, this was a dirty workspace change. But it's still wrong... putting it at the rust sbd until we're ready to properly switch to cloudflare.

```suggestion
pub const DEF_SIGNAL_URL: &str = "wss://sbd-0.main.infra.holo.host";
```